scrape.py

```diff
@@ -2,26 +2,36 @@
 from bs4 import BeautifulSoup
 
 
+def url_builder():
+    username = input("Enter user name: ")
+    url = 'https://www.codewars.com/users/' + username
+    return url
+
+
 def fetch(url):
     r = requests.get(url)
```

Comment:
I think it would be better to check if the user account exist over here instead of traversing through the code below. Just check what kind of status code the requests object is returning using if-else

Reply:
@palash25 Yeah, that's a good idea. I'll check for the status code.

Reply:
@palash25 Okay, I made the changes. Have a look at the code.